tests/playwright/propTests/labelVisibilityPropTest.js

```diff
@@ -0,0 +1,51 @@
+import { breakpoints } from '../constants/breakpoints';
+
+const windowHeight = 100;
+
+const generateLabelVisibilityPropTests = () => {
+  const tests = [
+    {
+      name: 'labelVisibility: none (hidden)',
+      props: {
+        labelVisibility: 'none',
+      },
+    },
+    {
+      name: 'labelVisibility: xs (visible)',
+      props: {
+        labelVisibility: 'xs',
+      },
+    },
+  ];
+
+  Object.entries(breakpoints).map(([breakpointName, breakpointValue]) => {
+    tests.push({
+      name: `labelVisibility: ${breakpointName} (visible)`,
+      onBeforeTest: async (page) => {
```

Comment:
This is quite interesting idea I had to came up with. Some test cases might require to modify browser and things like that. And this is a possible way how to do it.